pinot-common/src/main/java/com/linkedin/pinot/common/metadata/segment/IndexLoadingConfigMetadata.java

```diff
@@ -31,9 +31,11 @@
 
   public static final String KEY_OF_LOADING_INVERTED_INDEX = "metadata.loading.inverted.index.columns";
   public static final String KEY_OF_SEGMENT_FORMAT_VERSION = "segment.format.version";
+  public static final String KEY_OF_ADDING_DEFAULT_COLUMNS = "adding.default.columns";
```

Comment:
Actually I think it's more like default columns which use default null value. BTW, the handler name is DefaultColumnHandler.